superdesk/default_settings.py

```diff
@@ -834,3 +834,6 @@ def local_to_utc_hour(hour):
 TANSA_LICENSE_KEY = env('TANSA_LICENSE_KEY')
 TANSA_CLIENT_BASE_URL = env('TANSA_CLIENT_BASE_URL', 'https://d02.tansa.com/tansaclient/')
 TANSA_PROFILES = {}
+
+# Enable ninjs to send all the fields for place in output.
+NINJS_PLACE_EXTENDED = True
```

Comment:
would make it `False` by default so we don't change behaviour

Reply:
I will set it to false before merging. I already mentioned in the first comment.